src/c++/perf_analyzer/main.cc

```diff
@@ -735,6 +738,11 @@ Usage(char** argv, const std::string& msg = std::string())
              "eg:--model-repository=/tmp/host/docker-data/model_unit_test.",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --verbose-csv: The csv files generated by PA will include "
+                   "additional information necessary for Model Analyzer.",
```

Comment:
Let's not couple the tools in our docs. Just say what it does, not that it is for MA

Reply:
removed mention of MA 👍